### PR TITLE
Social links correctness

### PR DESCRIPTION
**Pull request by carlosfaria94**
If member don't have social links added to his profile, don't show the icons.

### CODE REVIEW

File: src/app/components/members/Member.js [hunk 1]
```diff
@@ -4,17 +4,25 @@ import { Col } from 'react-bootstrap'
...
+    let mailLink
+    if (typeof this.props.mail !== 'undefined') {
+      mailLink = <a target='_blank' href={'mailto:' + this.props.mail} className='social-icon fa fa-envelope-o' />
+    }
+    let twitterLink
+    if (typeof this.props.twitter !== 'undefined') {
+      twitterLink = <a target='_blank' href={'https://twitter.com/' + this.props.twitter} className='social-icon fa fa-twitter' />
+    }
+    let gitHubLink
+    if (typeof this.props.github !== 'undefined') {
```
Comment (johnytiago) [inline src/app/components/members/Member.js]: Why don't you just use `if (this.props.github) { ....`?

Comment (carlosfaria94) [reply src/app/components/members/Member.js]: In my opinion, checking if it is undefined is more reliable, and more accurate for this case.
But, of course, the two ways work

Comment (johnytiago) [reply src/app/components/members/Member.js]: I apologize if I'm being dumb, but when will `this.props.github` be `undefined` or `null` but still pass the check?

I made this test: 
screen shot 2016-11-22 at 19 35 18

Is there any edge case I'm not considering?
Btw, what do you mean with more reliable and acurate? :)

`0` or `''` will still pass the test

Comment (carlosfaria94) [reply src/app/components/members/Member.js]: When I talk about reliable, I mention the fact if `a` is not defined this will throw a ReferenceError:

image

Comment (carlosfaria94) [reply src/app/components/members/Member.js]: But ok, I will do it that way @johnytiago 😃

Comment (johnytiago) [reply src/app/components/members/Member.js]: That will happen of course but in this case it won't since `this` is always there, always defined. The worst that can happen is '`cannot to read property github of undefined`' in case props is not set (and that I don't know if is even possible).
